.github/workflows/build_docs.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install -r requirements.txt
+        pip install -e .
```

Comment:
@wuhu This was the bug preventing the apidocs from building correctly. We go onto to change directory to `padl/docs`, and the apidocs when building expects to be able to `import padl`